gems/nokogiri/CVE-2021-3518.yml

```diff
@@ -0,0 +1,27 @@
+---
+gem: nokogiri
+cve: 2021-3518
+ghsa: v4f8-2847-rwm7
+url: https://nokogiri.org/CHANGELOG.html#security_8
```

Comment:
Use the permalink: https://nokogiri.org/CHANGELOG.html#1114-2021-05-14

Reply:
Done